.github/workflows/stackhpc-container-image-build.yml

```diff
@@ -1,328 +1,36 @@
 ---
```

Comment:
What's going on here?

Reply:
Oh, it so I could test it before merging. I needed the action to already exist in https://github.com/stackhpc/stackhpc-kayobe-config/actions
to be able to trigger it before merging . The idea was to move to a new file once it worked. Not sure if there is an easier way?

Reply:
perhaps add a workflow_dispatch trigger to the PR workflow?